### PR TITLE
Week 5 - Guess Who - Sofia A.W

### PR DESCRIPTION
**Pull request by Sofarz**
Finished the project, did not do any of the stretch goals.

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -2,6 +2,10 @@
 const board = document.getElementById('board')
```
Comment (M4L5) [inline code/script.js:2]: In this area some of the DOM's have semicolons. Just a heads up to double check and choose if you want to go with semicolons or not. I actually did the same in my code so I will also need to double check my own :)

File: code/script.js [hunk 2]
```diff
@@ -247,97 +252,160 @@ const generateBoard = () => {
...
+  console.log('secret value: ', secretValue);
+  if (secretValue === currentQuestion.value) {
+    filterCharacters(true);
+  } else 
+    filterCharacters(false);
 }
 
 // It'll filter the characters array and redraw the game board.
 const filterCharacters = (keep) => {
+  let category = currentQuestion.category
```
Comment (M4L5) [inline code/script.js:314]: This part is very clear and and the code is easy to read. :D

**Review by M4L5 — COMMENTED** [review]
An overall comment for the whole code is that I think it looks good. It's easy to read and follow. I can't see anything right now that I can give suggestions on how to do in another way or think about. I think you did a great job with everything going on this week. Someone is a cool coding mom! 👍 👍

File: code/script.js (hunk 1 above)
Comment (KarolCislowski) [reply code/script.js:2]: You need to check your VScode settings and search for semicolons and if you decide to have them then make Prettier(vscode's extension- if you don't have it you need to install it) add them automatically after each saves.

File: code/script.js [hunk 3]
```diff
@@ -247,97 +252,160 @@ const generateBoard = () => {
...
+  let attribute = currentQuestion.attribute
+  let value = currentQuestion.value
+
+  console.log('category: ', category);
```
Comment (KarolCislowski) [inline code/script.js:318]: remember to always clean your code from all console.log()s and commented out code blocks 🤓

**Review by KarolCislowski — APPROVED** [review]
This is a very nice project 🌈 

I know it was an extra point of this assignment to style it your way but remember that there is no another way to master CSS then practice it over and over again 🦾 

Remember that you can always come back to it when you have some spare time and add more functionalities to make it an even greater portfolio asset! 🦄

**Pull request closed without merging**